content/en/blog/_posts/2021-22-02-The-Evolution-of-Kubernetes-Dashboard/index.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: blog
+title: "The Evolution of Kubernetes Dashboard"
+date: 2021-02-22
+slug: the-evolution-of-kubernetes-dashboard
+---
+
+Authors: Marcin Maciaszczyk and Sebastian Florek, Fullstack Engineers at Kubermatic
+
+In October 2020, the Kubernetes Dashboard officially turned five. As main project maintainers, we barely could believe that so much time has passed since our very first commit to the project. However, looking back with a bit of nostalgia, we had to realize that quite a lot has happened since then. Now it’s due time to celebrate “our baby” with a short recap.
+
+## How It All Began
+
+The initial idea behind the Kubernetes Dashboard project was to provide the web interface for Kubernetes. We wanted to reflect the kubectl functionality through an intuitive web UI. The big main benefit from using the UI is obviously to be able to quickly see things that do not work as expected (monitoring and troubleshooting). Also, the Kubernetes Dashboard is a great starting point for users that are new to the Kubernetes ecosystem.
+
+The very [first commit](https://github.com/kubernetes/dashboard/commit/5861187fa807ac1cc2d9b2ac786afeced065076c) to the Kubernetes Dashboard was made by Filip Grządkowski from Google on 16th October 2015 – just a few months from the initial commit to the Kubernetes repository. Our initial commits go back to November 2015 ([Sebastian committed on 16 November 2015](https://github.com/kubernetes/dashboard/commit/09e65b6bb08c49b926253de3621a73da05e400fd); [Marcin committed on 23 November 2015](https://github.com/kubernetes/dashboard/commit/1da4b1c25ef040818072c734f71333f9b4733f55)). Since that time, we’ve become regular contributors to the project. For the next two years, we will be working closely with the Googlers, eventually becoming main project maintainers ourselves.
+
+<div class="thumbnail" style="width: 80%;margin: auto">
+  <a href="./first-ui.png"><img src="./first-ui.png" alt="The First Version of the User Interface"></a>
+  <div class="caption" style="text-align: center; padding-top: .5em">
+    The First Version of the User Interface
+  </div>
+</div>
+
+<div class="thumbnail" style="width: 80%;margin: auto; padding-top: 2em">
+  <a href="./along-the-way-ui.png"><img src="./along-the-way-ui.png" alt="Prototype of the New User Interface"></a>
+  <div class="caption" style="text-align: center; padding-top: .5em">
+    Prototype of the New User Interface
+  </div>
+</div>
+
+<div class="thumbnail" style="width: 80%;margin: auto; padding: 2em 0">
+  <a href="./current-ui.png"><img src="./current-ui.png" alt="The Current User Interface"></a>
+  <div class="caption" style="text-align: center; padding-top: .5em">
+    The Current User Interface
+  </div>
+</div>
```

Comment:
Hi

I'd prefer to see images defined in Markdown rather than done using rendered HTML.

We can help you make use of Markdown and also Hugo shortcodes so that the article doesn't define HTML directly.

Reply:
I have tried to use original markdown/hugo, but with the current theme it wouldn't really allow to center images + use nice caption for them. I am open to suggestions here.

Another option was to add a custom shortcode with custom styling that would allow to better handle image positioning and caption, but it would require more extensive changes outside of only the blog directory, so I was hesitant to do so.

Reply:
Is `imgproc` from Docsy good enough?

https://www.docsy.dev/docs/adding-content/shortcodes/#imgproc

Reply:
It's ok'ish I guess. It is quite limited and requires manually tweaking the size (width, height). Also the position argument `left|center|right` does not seem to work. I have tried to make it look as best as possible given the limited functionality.